src/modules/dashboard/components/SubscribedColoniesList/SubscribedColoniesList.tsx

```diff
@@ -57,7 +57,7 @@ const SubscribedColoniesList = () => {
                     <ColonyAvatar
                       colony={colony}
                       colonyAddress={colonyAddress}
-                      size="xs"
+                      size="s"
```

Comment:
I think we should stick to sizes define in Avatar component so in case of "s" it will be 34x34 instead of adding new case in css 36x36, maybe lets ask Karol if he will agree with that?